packages/mobile/scripts/run_e2e.sh

```diff
@@ -166,20 +160,29 @@ if [ $PLATFORM = "android" ]; then
       exit 1
     fi
 
-    echo "Starting the emulator"
-    $ANDROID_SDK_ROOT/emulator/emulator \
-      -avd $VD_NAME \
-      -no-boot-anim \
-      -noaudio \
-      -no-snapshot \
-      -netdelay $NET_DELAY \
-      ${CI:+-gpu swiftshader_indirect -no-window} \
-      &
-
-    echo "Waiting for device to connect to Wifi, this is a good proxy the device is ready"
-    until [ `adb shell dumpsys wifi | grep "mNetworkInfo" | grep "state: CONNECTED" | wc -l` -gt 0 ]
+    for ((i=1; i<=$WORKERS; i=i+1))
```

Comment:
does this mean that we are spinning up N emulators on the same machine? i didn't know it was possible to spin up multiple copies of the same emulator actually. we're not explicitly doing this for iOS - does that mean detox is doing that under the hood? i'm curious for why the difference in how we are handling workers for iOS/android

Reply:
For iOS Detox does a good job of handling launching multiple Simulators. For Android Detox can launch the emulators; however, in practice I have seen a lot of test flake letting Detox launch Android Emulators. Wix even advises launching the emulators with adb in their own blogpost: [How to execute Android UI tests on CI and stay alive](https://medium.com/wix-engineering/how-to-execute-android-ui-tests-on-ci-and-stay-alive-eb9089d88c1f).